addons/web_editor/static/src/js/editor/rte.summernote.js

```diff
@@ -1230,12 +1230,16 @@ var SummernoteManager = Class.extend(mixins.EventDispatcherMixin, {
             return;
         }
         data.__alreadyDone = true;
+        var res_model = data.$editable.data('oe-model');
+        var mailCompose = res_model === "mail.compose.message";
+        var noVideos = data.options.noVideos || mailCompose;
 
         var mediaDialog = new weWidgets.MediaDialog(this,
             _.extend({
-                res_model: data.$editable.data('oe-model'),
+                res_model: res_model,
                 res_id: data.$editable.data('oe-id'),
                 domain: data.$editable.data('oe-media-domain'),
+                noVideos: noVideos,
```

Comment:
I think it is wrong since if `data.options.noVideos` is `false` then you will ignore `mailCompose` since the `extend` will overridde the default values here with `data.options`.

Anyways, here you are in the code which handles a dialog opening demand. It is not supposed to ignore the demands and change them. The code which really should have the control is the one making that demand, which is the `showImageDialog` function.

Reply:
All right.

Reply:
Updated with commit reference.